.travis.yml

```diff
@@ -1,29 +1,29 @@
 language: python
```

Comment:
Could you add a small matrix in here to test both Python 2 and 3?

Reply:
See if that's right

Reply:
Looks right but Travis will be quick to tell us otherwise :game_die: